integration/container/list_test.go

```diff
@@ -13,6 +14,37 @@ import (
 	is "gotest.tools/v3/assert/cmp"
 )
 
+func containerListContainsName(containerList []containertypes.Summary, name string) bool {
+	for _, ctr := range containerList {
+		if ctr.ID == name {
+			return true
+		}
+	}
+	return false
+}
+
+func TestList(t *testing.T) {
```

Comment:
I just noticed that we have both a `list_test.go` and  a `ps_test.go`, both with tests for listing; we should probably merge them; https://github.com/moby/moby/blob/126d4bc65b37353121250a93fea896b3a9364a0d/integration/container/ps_test.go#L14

Reply:
On that matter (but definitely out of scope for this PR); there's a bunch of old tests in integration-cli that should also be moved to the `integration/` tests, for example; https://github.com/moby/moby/blob/126d4bc65b37353121250a93fea896b3a9364a0d/integration-cli/docker_api_containers_test.go#L37

Reply:
I merged `ps_test.go` into `list_test.go` ... good catch!